### DISCUSSION TITLE
Line break in a table cell from an imported Markdown

### DISCUSSION THREAD

**Discussion by 411x** [outline/outline#6664] [Ideas]
When importing a .md file with a table with cells containing line breaks, the content does not apply the line breaks.
I haven't found any documentation on your site about this.
I found [this discussion ](https://github.com/outline/rich-markdown-editor/issues/206#issuecomment-678686015) and tried the `<br/>` but it doesn't seem to be interpreted by the parser.

I import large documents and it's a massive waste of time if I have to manually ctrl+enter.

**Comment by tommoor** [comment]
Unfortunately there is no standard for supporting newlines in tables in Markdown. We support newlines with an escaped `\\n` character – you can create a table in Outline and export as Markdown to see the format.

**Reply by 411x** [reply]
I've tried.

I made a table in outline, with a cell containing several manually made line breaks.

I did "copy markdown", then pasted it back into the document and the line breaks are not applied.

**Reply by 411x** [reply]
Ok if i add \n, it works.
